fdt.cc

```diff
@@ -918,6 +919,7 @@ node::node(text_input_buffer &input,
 		{
 			input.next_token();
 			omit_if_no_ref = true;
+			tree.needs_garbage_collection();
```

Comment:
Method with side effects should start with a verb.  Please call this `set_needs_garbage_collection`